charts/incubator/cryptpad/templates/common.yaml

```diff
@@ -0,0 +1,16 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* Append the configMap volume to the volumes */}}
+{{- define "cryptpad.configVolume" -}}
+enabled: "true"
+mountPath: "/cryptpad/config/config.js"
+subPath: "config.js"
+type: "custom"
+volumeSpec:
+  configMap:
+    name: {{ template "common.names.fullname" . }}-config
+{{- end -}}
+{{- $_ := set .Values.persistence "cryptpad-config" (include "cryptpad.configVolume" . | fromYaml) -}}
+
+{{ include "common.all" . }}
```

Comment:
I am not completely sure if this is idiomatic, but since I did have to use a configMap for this application to work I copy-pasted this from https://github.com/k8s-at-home/charts/blob/master/charts/stable/mosquitto/templates/common.yaml.

Reply:
This looks fine to me :D